fix path to stereo_chemical_props.txt when AlphaFold is installed with EasyBuild
author: Kenneth Hoste (HPC-UGent)
diff --git a/alphafold/common/residue_constants.py b/alphafold/common/residue_constants.py
index 318b44a..fd840e3 100644
--- a/alphafold/common/residue_constants.py
+++ b/alphafold/common/residue_constants.py
@@ -19,6 +19,7 @@ import functools
 from typing import Mapping, List, Tuple
 
 import numpy as np
+import os
 import tree
 
 # Internal import (35fd).
@@ -403,7 +404,7 @@ def load_stereo_chemical_props() -> Tuple[Mapping[str, List[Bond]],
     residue_bond_angles: dict that maps resname --> list of BondAngle tuples
   """
   stereo_chemical_props_path = (
-      'alphafold/common/stereo_chemical_props.txt')
+      os.path.join(os.getenv('EBROOTALPHAFOLD', 'alphafold/common'), 'stereo_chemical_props.txt'))
   with open(stereo_chemical_props_path, 'rt') as f:
     stereo_chemical_props = f.read()
   lines_iter = iter(stereo_chemical_props.splitlines())
